### PR TITLE
Update README with sitemap Input example

### PR DESCRIPTION
**Pull request by mlobstein**
Update the README for kaleidescape and tivo to provide example usages of the sitemap element type 'Input'.

### CODE REVIEW

File: bundles/org.openhab.binding.kaleidescape/README.md [hunk 1]
```diff
@@ -430,4 +432,26 @@ then
     }
     kactions.sendKCommand("GET_CONTENT_DETAILS:" + z1_Music_AlbumHandle.state.toString + ":")
 end
+
+rule "Movie Search"
+when
+    Item z1_MovieSearch received update
+then
+    if (z1_MovieSearch.state != NULL && z1_MovieSearch.state.toString.length > 0) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.kaleidescape/README.md:440]: If `received update`, use `newState` instead of `z1_MovieSearch`.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.kaleidescape/README.md (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.kaleidescape/README.md]: ok, fixed on both readmes

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Review by kaikreuzer — APPROVED** [review]
Thanks!

**Pull request merged**